cl/corpus_importer/fixtures/tenn_court_fixture.json

```diff
@@ -0,0 +1,1284 @@
+[
+  {
+    "model": "people_db.position",
```

Comment:
I suspect we don't need the full fixture for tests. Can we slim it down considerably? The advantage of doing so is two-fold. One, it takes up less space in the repo. Two, when you're debugging later, it's easier to understand which parts matter.

Reply:
slimmed down